modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/OptionCategoryResourceImpl.java

```diff
@@ -146,10 +172,27 @@ private CPOptionCategory _updateOptionCategory(
 		CPOptionCategory cpOptionCategory =
 			_cpOptionCategoryService.getCPOptionCategory(id);
 
+		Map<String, String> titleMap = optionCategory.getTitle();
+
+		if ((cpOptionCategory != null) && (titleMap == null)) {
```

Comment:
I think cpOptionCategory is always not null because of
```
CPOptionCategory cpOptionCategory =
			_cpOptionCategoryService.getCPOptionCategory(id);
```